src/js/pensions/components/FinancialDisclosureDescription.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import Scroll from 'react-scroll';
+
+const scroller = Scroll.scroller;
+
+function scrollToPage(key) {
+  scroller.scrollTo(`${key}ScrollElement`, {
+    duration: 500,
+    delay: 2,
+    smooth: true,
+  });
+}
+
+export default function FinancialDisclosureDescription({ viewedPages, pageKeys }) {
+  const unViewedPages = pageKeys.filter(key => !viewedPages.has(key));
+  if (!unViewedPages.length) {
+    return null;
+  }
+
+  const jumpToFirst = () => scrollToPage(unViewedPages[0]);
+  let alertBody;
+  if (unViewedPages.some(key => key.includes('spouse'))) {
+    alertBody = (
+      <div className="usa-alert-body">
+        Does your spouse have any net worth, income, or expenses?
```

Comment:
I don't _think_ so, but I'm not positive. I think it's likely easier to direct them to the earlier new pages (which are the spouse ones) and they'll notice the later ones below the spouse pages when they complete them.